App.js

```diff
@@ -1,20 +1,32 @@
+import 'react-native-gesture-handler'
 import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { StyleSheet, Text, View, SafeAreaView } from 'react-native';
+import { NavigationContainer } from '@react-navigation/native'
+// provides a native React view that transitions between multiple colors in a linear direction.
+
+import DrawerNavigator from './navigation/DrawerNavigation';
+import LandingStackNavigator from './navigation/StackNavigation';
+const user = false; 
```

Comment:
I understand what you're doing with the user variable, but wouldnt it be better to set the user variable to context or redux and then change the variable within that rather than set the user variable to a global variable?